src/wasm2js.h

```diff
@@ -2841,7 +2841,7 @@ void Wasm2JSGlue::emitSpecialSupport() {
     assert(bytes == 8); // TODO: support 1, 2, 4 as well
     var view = new BigInt64Array(bufferView.buffer); // TODO cache
     ptr = (ptr + offset) >> 3;
-    var value = BigInt(valueLow >>> 0) | (BigInt(valueHigh >>> 0) << BigInt(32));
+    var value = BigInt(valueLow >>> 0) | (BigInt(valueHigh >>> 0) << 32n);
```

Comment:
I'm not sure this will work in emscripten - we need to verify that the parser there can handle bigint literals. It failed on that a while ago, but maybe works now.

Checking `./tests/runner.py wasm2js1 wasm2js*.test_loop` should be enough to verify things are ok.

Reply:
> ./tests/runner.py wasm2js2 ?

Hmm, I don't see this folder

Reply:
Yes, the emscripten test suite is in the emscripten repo.

Reply:
No, I don't try it. I have some issue with building emscripten from source. Could somebody check this? Or I could revert only this changes